docs/luigi-container-api.md

```diff
@@ -16,7 +16,8 @@ meta -->
 
 # Luigi Container API
 
-This document outlines the parameters provided by the Luigi Container.
+This document outlines the parameters provided by the Luigi Container.</br>
+In addition you can use standard `addEventListener` function to react on events emmitted by the luigi container. The list of events and their meaning can be found [here](https://github.com/SAP/luigi/blob/main/container/src/constants/communication.ts).
```

Comment:
Perhaps capitalized version would be more suitable? This pattern is used in the sentence above:
'Luigi Container'

Reply:
thx